pkgs/by-name/la/lazymc/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib
+, rustPlatform
+, fetchFromGitHub
+, nix-update-script
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lazymc";
+  version = "0.2.10";
+
+  src = fetchFromGitHub {
+    owner = "timvisee";
+    repo = "lazymc";
+    rev = "v${version}";
+    hash = "sha256-IObLjxuMJDjZ3M6M1DaPvmoRqAydbLKdpTQ3Vs+B9Oo=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "minecraft-protocol-0.1.0" = "sha256-vSFS1yVxTBSpx/ZhzA3EjcZyOWHbmoGARl0eMn1fJ+4=";
+    };
+  };
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    description = "Remote wake-up daemon for minecraft servers";
+    homepage = "https://github.com/timvisee/lazymc";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ h7x4 dandellion ];
+    platforms = platforms.unix;
```

Comment:
```suggestion
```
that's the default for buildRustPackage

Reply:
I've been back and forth about this on quite a few PRs now, it would be nice to come to a proper understanding of what's going on with `buildRustPackage`s `meta.platforms`. As I've understood it until now, `buildRustPackage` set all platforms that rust can build to as its default. This includes stuff like web assembly which is (probably) not a target for this particular package. Would you mind having a look at this?

https://github.com/NixOS/nixpkgs/blob/3e8293e8934f5870b2c5be84fe37d5b61deda6af/pkgs/build-support/rust/build-rust-package/default.nix#L153-L168

Reply:
Your understanding is correct, exa does not have platforms set and this is what it gets set to:

```
nix-repl> exa.meta.platforms
[ "x86_64-darwin" "i686-darwin" "aarch64-darwin" "i686-freebsd13" "x86_64-freebsd13" "x86_64-solaris" "aarch64-linux" "armv7l-linux" "i686-linux" "mipsel-linux" "mips64el-linux" "powerpc64-linux" "powerpc64le-linux" "riscv64-linux" "s390x-linux" "x86_64-linux" "aarch64-netbsd" "armv7l-netbsd" "i686-netbsd" "powerpc-netbsd" "x86_64-netbsd" "i686-openbsd" "x86_64-openbsd" "i686-windows" "x86_64-windows" "armv7a-darwin" "armv5tel-linux" "armv6l-linux" "armv7a-linux" "m68k-linux" "riscv32-linux" "armv6l-netbsd" "x86_64-redox" "wasm32-wasi" ]
```